packages/zendesk-adapter/src/config.ts

```diff
@@ -1649,13 +1649,13 @@ export const DEFAULT_TYPES: ZendeskApiConfig['types'] = {
     transformation: {
       idFields: ['&section_id', 'title'],
       fileNameFields: ['&section_id', 'title'],
-      standaloneFields: [{ fieldName: 'translations' }],
+      standaloneFields: [{ fieldName: 'translations' }, { fieldName: 'attachments' }],
       sourceTypeName: 'articles__articles',
       fieldsToHide: FIELDS_TO_HIDE.concat(
         { fieldName: 'id', fieldType: 'number' },
         { fieldName: 'position', fieldType: 'number' },
       ),
-      fieldTypeOverrides: [{ fieldName: 'id', fieldType: 'number' }, { fieldName: 'author_id', fieldType: 'unknown' }],
+      fieldTypeOverrides: [{ fieldName: 'id', fieldType: 'number' }, { fieldName: 'author_id', fieldType: 'unknown' }, { fieldName: 'attachments', fieldType: 'list<unknown>' }],
```

Comment:
nit:
```suggestion
      fieldTypeOverrides: [
        { fieldName: 'id', fieldType: 'number' },
        { fieldName: 'author_id', fieldType: 'unknown' },
        { fieldName: 'attachments', fieldType: 'list<unknown>' },
      ],
```
(also can we make these not unknown, e.g. make `author_id` a number/string?